[2024-02-04T17:31:02.188+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: modeling_dag.consume_task scheduled__2024-02-04T17:30:00+00:00 [queued]>
[2024-02-04T17:31:02.194+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: modeling_dag.consume_task scheduled__2024-02-04T17:30:00+00:00 [queued]>
[2024-02-04T17:31:02.194+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-02-04T17:31:02.202+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): consume_task> on 2024-02-04 17:30:00+00:00
[2024-02-04T17:31:02.208+0000] {standard_task_runner.py:57} INFO - Started process 4469 to run task
[2024-02-04T17:31:02.210+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'modeling_dag', 'consume_task', 'scheduled__2024-02-04T17:30:00+00:00', '--job-id', '1811', '--raw', '--subdir', 'DAGS_FOLDER/modeling.py', '--cfg-path', '/tmp/tmp3ye8q2yp']
[2024-02-04T17:31:02.212+0000] {standard_task_runner.py:85} INFO - Job 1811: Subtask consume_task
[2024-02-04T17:31:02.245+0000] {task_command.py:410} INFO - Running <TaskInstance: modeling_dag.consume_task scheduled__2024-02-04T17:30:00+00:00 [running]> on host 93456042b127
[2024-02-04T17:31:02.312+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='modeling_dag' AIRFLOW_CTX_TASK_ID='consume_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-04T17:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-04T17:30:00+00:00'
[2024-02-04T17:31:02.337+0000] {logging_mixin.py:150} INFO - 받은 메시지 [[30.919680000000003, 6.33159, 24.78339, 150.66378, 24.98328, 1255.37814]]
[2024-02-04T17:31:02.343+0000] {python.py:183} INFO - Done. Returned value was: [[30.919680000000003, 6.33159, 24.78339, 150.66378, 24.98328, 1255.37814]]
[2024-02-04T17:31:02.360+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=modeling_dag, task_id=consume_task, execution_date=20240204T173000, start_date=20240204T173102, end_date=20240204T173102
[2024-02-04T17:31:02.382+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-02-04T17:31:02.403+0000] {taskinstance.py:2653} INFO - 2 downstream tasks scheduled from follow-on schedule check
